service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -151,96 +149,87 @@ public GetTariffServiceDto editTariffService(EditTariffServiceDto dto, Integer i
+        }
+    }
 
-        Service service = Service.builder()
-            .basePrice(dto.getPrice())
-            .commission(dto.getCommission())
-            .fullPrice(dto.getPrice() + dto.getCommission())
-            .capacity(dto.getCapacity())
-            .createdAt(LocalDate.now())
-            .courier(courier)
-            .createdBy(user.getRecipientName() + " " + user.getRecipientSurname())
-            .serviceTranslations(dto.getServiceTranslationDtoList()
-                .stream().map(serviceTranslationDto -> ServiceTranslation.builder()
-                    .description(serviceTranslationDto.getDescription())
-                    .descriptionEng(serviceTranslationDto.getDescriptionEng())
-                    .name(serviceTranslationDto.getName())
-                    .nameEng(serviceTranslationDto.getNameEng())
-                    .build())
-                .collect(Collectors.toList()))
-            .build();
-        service.getServiceTranslations().forEach(serviceTranslation -> serviceTranslation.setService(service));
-        return service;
+    private TariffsInfo getTariffsInfoById(long tariffId) {
+        Optional<TariffsInfo> tariffsInfo = tariffsInfoRepository.findById(tariffId);
+        if (tariffsInfo.isPresent()) {
+            return tariffsInfo.get();
+        } else {
+            throw new NotFoundException(ErrorMessage.TARIFF_NOT_FOUND + tariffId);
+        }
     }
 
-    @Override
-    public List<GetServiceDto> getService() {
-        return serviceTranslationRepository.findAll()
-            .stream()
-            .map(this::getService)
-            .collect(Collectors.toList());
+    private Optional<Service> getServiceByTariffsInfoId(long tariffId) {
+        return serviceRepository.findServiceByTariffsInfoId(tariffId);
     }
 
-    private GetServiceDto getService(ServiceTranslation serviceTranslation) {
-        return GetServiceDto.builder()
-            .courierId(serviceTranslation.getService().getCourier().getId())
-            .description(serviceTranslation.getDescription())
-            .descriptionEng(serviceTranslation.getDescriptionEng())
-            .price(serviceTranslation.getService().getBasePrice())
-            .capacity(serviceTranslation.getService().getCapacity())
-            .name(serviceTranslation.getName())
-            .nameEng(serviceTranslation.getNameEng())
-            .commission(serviceTranslation.getService().getCommission())
-            .fullPrice(serviceTranslation.getService().getFullPrice())
-            .id(serviceTranslation.getService().getId())
-            .createdAt(serviceTranslation.getService().getCreatedAt())
-            .createdBy(serviceTranslation.getService().getCreatedBy())
-            .editedAt(serviceTranslation.getService().getEditedAt())
-            .editedBy(serviceTranslation.getService().getEditedBy())
-            .build();
+    @Override
+    public GetServiceDto getService(long tariffId) {
+        getTariffsInfoById(tariffId);
```

Comment:
Why do we need it?
The result of `getTariffsInfoById(tariffId)` is not stored to the variable and never used